components/common/hocs/cloud/with_get_cloud_subscription.tsx

```diff
@@ -22,12 +22,14 @@ interface UsedHocProps {
 function withGetCloudSubscription<P>(WrappedComponent: ComponentType<P>): ComponentType<any> {
     return class extends React.Component<P & UsedHocProps> {
         async componentDidMount() {
-            const {subscription, actions: {getCloudSubscription}, isCloud, userIsAdmin} = this.props;
+            // if not is cloud, not even try to destructure values from props, just return
+            if (!this.props.isCloud) {
```

Comment:
Requesting we update the test

Reply:
the test already cover the scenario where is not cloud.